### ISSUE TITLE
Printing rants

### ISSUE THREAD

**Issue by DeveloperACE** [devRant/devRant#374]
I read this rant recently and really wanted to print it:
https://devrant.com/rants/1315593/as-a-developer-sometimes-you-hammer-away-on-some-useless-solo-side-project-for-a

So I copied the link to the devrant website (since there’s no option to print from the app). Just after selecting the printer I saw this:
![ee2ff622-d7a9-4912-9876-2dff07337025](https://user-images.githubusercontent.com/17362949/38222008-519f7820-3697-11e8-80ec-ec2335756030.png)

So basically I wanted to put in a feature request for the devrant web app (and/or mobile app) for something like a print-optimized stylesheet.
